From randy.dunlap@oracle.com  Sun Jan 25 15:22:23 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Tue, 20 Jan 2009 16:29:13 -0800
Subject: driver-core: fix kernel-doc parameter name
To: lkml <linux-kernel@vger.kernel.org>
Cc: gregkh <greg@kroah.com>, Mark McLoughlin <markmc@redhat.com>
Message-ID: <20090120162913.fffe3305.randy.dunlap@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

Fix function parameter name in kernel-doc:

Warning(linux-next-20090120//drivers/base/core.c:1289): No description found for parameter 'dev'
Warning(linux-next-20090120//drivers/base/core.c:1289): Excess function parameter 'root' description in 'root_device_unregister'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Acked-by: Mark McLoughlin <markmc@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/base/core.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/base/core.c
+++ b/drivers/base/core.c
@@ -1280,7 +1280,7 @@ EXPORT_SYMBOL_GPL(__root_device_register
 
 /**
  * root_device_unregister - unregister and free a root device
- * @root: device going away.
+ * @dev: device going away
  *
  * This function unregisters and cleans up a device that was created by
  * root_device_register().
